jentool/__init__.py

```diff
@@ -94,6 +94,12 @@ def _parser():
     parser_nodes_list = subparsers.add_parser(
         'nodes-list', help='List all nodes')
     parser_nodes_list.set_defaults(func=nodes_list)
+    # failing jobs
+    parser_failing_jobs = subparsers.add_parser(
+        'failing-jobs', help='List failing jobs')
```

Comment:
please rename to `jobs-failing`. That's the used naming schema:
```
$ jentool -h
<snipped>
sub-command help:
  {jobs-disable,jobs-delete,jobs-list,jobs-config,jobs-copy,nodes-list}
    jobs-disable        Disable the given job(s)
    jobs-delete         Delete the given job(s)
    jobs-list           List the given job(s)
    jobs-config         Get config(s) for the given job(s)
    jobs-copy           Copy the given job(s)
```

Reply:
Done